android/expoview/src/main/java/host/exp/exponent/modules/ClearExperienceData.java

```diff
@@ -1,39 +0,0 @@
-// Copyright 2015-present 650 Industries. All rights reserved.
```

Comment:
Should there be a .kt complement of this file?

Reply:
Nope, this file was unused so I removed it. Checked for usage via grep.